Lib/codeop.py

```diff
@@ -130,7 +130,7 @@ def __init__(self):
         self.flags = PyCF_DONT_IMPLY_DEDENT
 
     def __call__(self, source, filename, symbol):
-        codeob = compile(source, filename, symbol, self.flags, 1)
+        codeob = compile(source, filename, symbol, self.flags, True)
```

Comment:
https://docs.python.org/3.9/library/functions.html#compile

> The argument optimize specifies the optimization level of the compiler; the default value of -1 selects the optimization level of the interpreter as given by -O options. Explicit levels are 0 (no optimization; __debug__ is true), 1 (asserts are removed, __debug__ is false) or 2 (docstrings are removed too).

Reply:
It is `dont_inherit`, not `optimize`.

Reply:
Oops, sorry.